### PR TITLE
Testing: scripts to setup nodeport public IP

### PR DESCRIPTION
**Pull request by csrwng**
Scripts to setup a bastion machine with a public IP that can forward
nodeports to a management cluster.

### CODE REVIEW

File: test/hack/nodeport-ip-aws/setup.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+   --group-id "${MASTERSGID}" \
+   --protocol tcp \
+   --port 32000-32767 \
+   --source-group "${SGID}"
+
+# Find IP of a master machine
+MASTERIP="$(aws ec2 describe-instances --filters Name=tag:Name,Values=${INFRAID}-master-0 | jq -r '.Reservations[].Instances[0].PrivateIpAddress')"
+
+# Enable IP forwarding on bastion machine
+ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i "${SSHPRIVATEKEY}" "ec2-user@${PUBLICIP}" \
```
Comment (alvaroaleman) [inline test/hack/nodeport-ip-aws/setup.sh:59]: Can we not do that? How a private key is provided is IMHO an env config that is outside of the scope of this script. The main issue with the current approach is that it can not work for encrypted ssh keys that are provided through a ssh agent and always require configs if the ssh key is not in `${HOME}/.ssh/id_rsa` (which is something the `ssh` cli defaults to anyways)

Comment (csrwng) [reply test/hack/nodeport-ip-aws/setup.sh:59]: Sure, will remove the need for a private key and update the instructions to do an ssh-add if needed.
